app/types/declarations.d.ts

```diff
@@ -19,7 +19,7 @@ declare module 'react-native-animated-ellipsis';
 declare module 'react-native-html-to-pdf';
 declare module 'react-native-keychain';
 declare module 'react-native-receive-sharing-intent';
-declare module 'react-native-swipeable';
+declare module 'react-native-gesture-handler';
```

Comment:
```suggestion
```

As I mentioned in an earlier comment, this declaration is not needed. The library exports its own TypeScript definitions.